apps/shared/avifjpeg.c

```diff
@@ -1172,7 +1172,7 @@ static avifBool avifJPEGReadInternal(FILE * f,
 
     if (avif->xmp.size > 0 && ignoreXMP) {
         // Clear XMP in case we read it for something else (like gain map).
-        avifImageSetMetadataXMP(avif, NULL, 0);
+        AVIF_CHECK(avifImageSetMetadataXMP(avif, NULL, 0) == AVIF_RESULT_OK);
```

Comment:
(No action required) Yeah... I didn't add that one because it shouldn't fail and I thought it was fine even if it did, but it doesn't hurt to add the check.

Reply:
Ack